File: pgm/ABC_Dart/ABC_359_A.dart
// import 'dart:collection';
import 'dart:io';
// import 'dart:math';
// import 'dart:typed_data';

void main() {
  stdin.readLineSync()!;
  const String Takahashi = "Takahashi";
  const int zero = 0;
  String? S;
  int cnt = zero;
  while ((S = stdin.readLineSync()) != null) {
    if (S == Takahashi) {
      cnt++;
    }
  }
  print(cnt);
  exit(0);
}

File: pgm/ABC_Kotlin/ABC_359_A.kt
// import java.time.*
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.util.StringTokenizer

// import kotlin.math.*

const val Takahashi: String = "Takahashi"
const val zero: Int = 0

fun main() {
    readln()
    var S: String?
    var cnt: Int = zero
    while (readlnOrNull().also { S = it } != null) {
        if (S == Takahashi) {
            cnt++
        }
    }
    println(cnt)
    kotlin.system.exitProcess(0)
}